docs/changehistory/NextVersion.md

```diff
@@ -2,3 +2,23 @@
 publish: false
 ---
 # NextVersion
+
+1. Fail pull/merge/push when conflict handler detect data conflicts
+
+## Fail pull/merge/push when conflict handler detect data conflicts
+
+`BriefcaseDb.pushChanges()` will fail with following error, if their was data conflict detected.
+
+> _`UPDATE/DELETE before value do not match with one in db or CASCADE action was triggered`_
+
+### What is data conflict?
+
+Data conflict come in two flavours,
+
+1. Dirty reads happen when current briefcase update value of a property `P` from `X` to `Y`. But incoming changeset has property `P` with value `K` instead of `X`.
```

Comment:
I don't understand this sentence, could you rephrase it please?

Reply:
I will fix it. But it means the old values do not match for the changed row. The change is of type update, so it has old and new values. When the current db does not have the old values for a row then it causes a data conflict. 

Let's say the user reads x=1 and changes it to 2.  But when he does pull/merge another user has already changed x from 1 to 9. When applying the change for the current briefcase x=2 does not match the old value of x=1 and so it fails.

Reply:
Am I misunderstanding or could this simply be stated as "two briefcases attempt to modify the same property"?

Reply:
yes that is correct.

Reply:
I have edited NextVersion.md for clarity. Please resolve this comment if you're ok with those changes.